cmd/mt-http-gw/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/grafana/metrictank/logger"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"runtime"
+)
+
+var (
+	version       = "(none)"
+	showVersion   = flag.Bool("version", false, "print version string")
+	metrictankUrl = flag.String("metrictank-url", "", "the url of the metrictank instance to proxy requests to")
+	bindAddress   = flag.String("bind-address", ":8080", "the address to bind to")
+)
+
+func init() {
+	formatter := &logger.TextFormatter{}
+	formatter.TimestampFormat = "2006-01-02 15:04:05.000"
+	log.SetFormatter(formatter)
+	log.SetLevel(log.InfoLevel)
+}
+
+func main() {
+	flag.Usage = func() {
+		fmt.Println("mt-http-gw")
```

Comment:
I think the initial plan was for this to be `mt-http-ingest`, but gw (gateway) is probably a more accurate description.  Anyone got a better alternative?

Reply:
gw seems better. perhaps we can do mt-gw because it's shorter and because in the future maybe we'll add non-http protocols, and even if we don't, gateway kindof implies http anyway

Reply:
`mt-gateway` maybe?  Without the `http` context I feel like it's not super apparent what `gw` means

Reply:
i like mt-gateway :+1:

Reply:
renamed in 061ecb22b67a4efff93a2e019ea66feb6198864b